test/integration/livekit-webhook-handler.spec.ts

```diff
@@ -306,4 +334,141 @@ test('POST /livekit-webhook', ({ components, spyComponents }) => {
       })
     })
   })
+
+  describe('when the room is a community voice chat room', () => {
+    const communityId = 'test-community-123'
+    const moderatorAddress = '0x1234567890123456789012345678901234567890'
+    const memberAddress = '0x1234567890123456789012345678901234567891'
+
+    beforeEach(async () => {
+      webhookEvent.room.name = `voice-chat-community-${communityId}`
+      await components.voiceDB.createCommunityVoiceChatRoom(webhookEvent.room.name, moderatorAddress)
+      await components.voiceDB.joinUserToCommunityRoom(moderatorAddress, webhookEvent.room.name, true)
+      await components.voiceDB.joinUserToCommunityRoom(memberAddress, webhookEvent.room.name, false)
+    })
+
+    afterEach(async () => {
+      await components.voiceDB.deleteCommunityVoiceChat(webhookEvent.room.name)
+    })
+
+    describe('when a participant leaves', () => {
+      beforeEach(() => {
+        webhookEvent.event = 'participant_left'
+        webhookEvent.participant.identity = memberAddress
+        webhookEvent.participant.disconnectReason = DisconnectReason.CLIENT_INITIATED
+      })
+
+      it('should handle community participant left event', async () => {
+        const response = await makeRequest(components.localFetch, '/livekit-webhook', {
+          method: 'POST',
+          headers: {
+            Authorization: 'Bearer aToken',
+            'Content-Type': 'application/json'
+          },
+          body: 'aBody'
+        })
+
+        expect(response.status).toBe(200)
+        expect(handleParticipantLeftSpy).toHaveBeenCalledWith(
+          memberAddress,
+          webhookEvent.room.name,
+          DisconnectReason.CLIENT_INITIATED
+        )
+      })
+
+      describe('when the last moderator leaves', () => {
+        beforeEach(() => {
+          webhookEvent.participant.identity = moderatorAddress
+          spyComponents.livekit.deleteRoom.mockResolvedValue(undefined)
+        })
+
+        it('should handle moderator leaving and destroy room if no other moderators', async () => {
+          const response = await makeRequest(components.localFetch, '/livekit-webhook', {
+            method: 'POST',
+            headers: {
+              Authorization: 'Bearer aToken',
+              'Content-Type': 'application/json'
+            },
+            body: 'aBody'
+          })
+
+          expect(response.status).toBe(200)
+          expect(handleParticipantLeftSpy).toHaveBeenCalledWith(
```

Comment:
These test cases are great! But they're not asserting over the DB. As these are the only tests that would check that the DB is correctly updated when a user joins or leaves, it would be great to perform these checks of the DB. This can be done in another PR if you want to.